podman.hcl

```diff
@@ -64,13 +65,27 @@ version "4.4.1" "4.4.4" "4.5.0" "4.5.1" "4.6.0" "4.6.1" "4.6.2" "4.7.0" "4.7.1"
 
   platform "linux" {
     source = "https://github.com/containers/podman/releases/download/v${version}/podman-remote-static-linux_${arch}.tar.gz"
+    env = {
+      "CONTAINERS_CONF": "${HERMIT_ENV}/.hermit/podman/containers.conf",
+      "CONTAINER_HOST": "unix://${HOME}/.local/share/containers/podman/machine/qemu/podman.sock",
+    }
 
     on "unpack" {
       rename {
         from = "${root}/bin/podman-remote-static-linux_${arch}"
         to = "${root}/podman"
       }
     }
+
+    on "install" {
+      mkdir {
+        dir = "${HERMIT_ENV}/.hermit/podman"
```

Comment:
Unfortunately this won't do what you hope. Install hooks only trigger once per machine-wide install, so updating the Hermit environment isn't reliable.

Reply:
ah. ok. so this should be on activation then?

Reply:
There's no real way to do what you want reliably in Hermit :(

Activation triggers when `. ./bin/activate-hermit` is called, but it won't trigger when a binary is executed directly via `./bin/<foo>`, so it's not reliable. It's unfortunate.

Reply:
it only needs to be done once. if it's done immediately after install then all is good. if there is no way to do this, then the only solution is to replace podman with a shell script that takes care of the initialization if not done.

thoughts?